components/brave_new_tab_ui/stories/default/data/backgroundWallpaper.ts

```diff
@@ -41,21 +42,30 @@ export const backgroundWallpapers = (function (images: NewTab.BackgroundWallpape
     })
   }
 
+  for (const gradient of gradientColors) {
```

Comment:
If you like, you can use `reduce` here:

```suggestion
  staticImages = gradientColors.reduce((prev, gradient) => ({
      ...prev,
      [gradient]: {
          type: 'gradientColor',
          wallPaperGradientColor: gradient
      }
  }), staticImages)
```

Reply:
Cool, Done!